Update to PPP (KN) (#230)
AY2122S1-CS2103T-T10-2/tp: docs/team/lowkangn.md

* **New Feature**: `View a Participant's details`
    * What it does: Allows users to view the detailed information of a participant in the list, including the list of 
      events the participant is attending.
    * Justification: After many events and participants have been added to Managera, users may want to recall which 
    events a certain participant is enrolled in to make plans for their upcoming schedules.

* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/?search=AY2122S1-CS2103T-T10-2&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2021-09-17&tabOpen=true&tabType=authorship&tabAuthor=lowkangn&tabRepo=AY2122S1-CS2103T-T10-2%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code~other&authorshipIsBinaryFileTypeChecked=false)

* **Project management**:
    * Handled the Github issue tracker and added/kept track of issues to be resolved.
    * Ensured that deadlines were met and deliverables were up to standard.

* **Enhancements to existing features**:

>     * I did not provide enhancements to any existing features.

Reviewer: Sounds a bit out of place to use a personal pronoun here
```suggestion
    * No enhancements are provided to any existing features
```
